03_building_and_packaging/pip_demo.md

```diff
@@ -0,0 +1,87 @@
+# `pip`: Python Packaging Tool (Notes)
+
+**Note**: Maybe run everything in a fresh Docker container. I built the container `jaustar/spack-package-tutorial-base` locally for myself. It contains installed/downloaded Spack, that was not set up yet, i.e., one needs to run `spack compiler find` and `spack external find`. `clingo` needs to be downloaded first as well.
+
+## 1. Installing packages with `pip`
+
+Installing a package
+
+```bash
+pip install micro-manager-precice
+```
+
```

Comment:
In the very beginning you could explain how to get `pip` in the first place and that `pip` has versions as well.
And maybe that there might be a difference between `pip` and `pip3`.